src/components/TheToolPanel.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="custom-tool-panel-container btn-toggle-box pa-0">
-    <v-btn-toggle class="flex-column">
+    <v-btn-toggle class="flex-column custom-tool-bar" style="border-radius: 0 !important;">
```

Comment:
can this style get moved to the class, and avoid !important if possible - try increasing the specificity, look at dev tools to see what class/styles are overriding it